{"uuid":"85c15c0d-e209-431c-a396-2b75bc0a6225","historyId":"a4fac7127e33025a97f1e886dbd43c66","fullName":"com.thetestingacademy.web.tests.LoginTest.testVWOLogin_Negative","labels":[{"name":"package","value":"com.thetestingacademy.web.tests.LoginTest"},{"name":"testClass","value":"com.thetestingacademy.web.tests.LoginTest"},{"name":"testMethod","value":"testVWOLogin_Negative"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"SeleniumPOM"},{"name":"subSuite","value":"com.thetestingacademy.web.tests.LoginTest"},{"name":"host","value":"Sanjays-MBP"},{"name":"thread","value":"4255@Sanjays-MBP.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"critical"}],"links":[],"name":"testVWOLogin_Negative","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\nExpecting not blank but was: \"\"","trace":"java.lang.AssertionError: \nExpecting not blank but was: \"\"\n\tat com.thetestingacademy.web.tests.LoginTest.testVWOLogin_Negative(LoginTest.java:22)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\n\tat org.testng.TestNG.run(TestNG.java:1082)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","description":"Verify the INValid Credential have expected outcome","steps":[],"attachments":[],"parameters":[],"start":1692555147640,"stop":1692555154163}